src/resources/js/admin/tickets-commerce-settings.js

```diff
@@ -250,6 +251,43 @@ tribe.tickets.admin.commerceSettings = {};
 			obj.disconnectButton.on( 'click', obj.handleDisconnectClick );
 		}
 
+		$document.on( 'click', obj.selectors.webHookSetUpLink, function( evt ) {
```

Comment:
The `obj.selectors.webHookSetUpLink` should have some reference to Stripe.

Reply:
Added a link to Stripe's API docs for Webhooks